src/node/NodeOptions.hs

```diff
@@ -222,6 +207,25 @@ addrNodeOption =
         metavar "HOST:PORT" <>
         help "Identifier of a node in a Kademlia network"
 
+tlsParamsOption :: Parser TlsParams
```

Comment:
> Looks like TLS params are added to light wallet options, but never used. Is it intentional?

I was not sure how light wallet will be used. As it is not used by Daedalus atm I think we can remove it. In this case light wallet should not be installed on clients machine when daedalus is installed with installers. If it is installed, some users might want to use this instead of official Daedalus (ie, for hacking around) and they might get screwed if TLS is not on there as adversary can read the traffic there - and make some actions ? (not sure about this one)

So in the lack of understanding how light wallet works, what is its aim, and will it be installed on users machine or not I have left TLS on.